### PR TITLE
Configure multiple topics per route

### PR DESCRIPTION
**Pull request by schmika**

### CODE REVIEW

File: framework/src/main/java/com/github/dbmdz/flusswerk/framework/rabbitmq/Route.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import com.github.dbmdz.flusswerk.framework.model.Message;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+
+/** Flusswerk-specific abstraction, collection of topics/queues. No equivalent in RabbitMQ. */
+public class Route {
+  private String name;
```
Comment (bitzl) [inline framework/src/main/java/com/github/dbmdz/flusswerk/framework/rabbitmq/Route.java:12]:
```suggestion
  private final String name;
```

File: framework/src/main/java/com/github/dbmdz/flusswerk/framework/rabbitmq/Route.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  private List<Topic> topics;
```
Comment (bitzl) [inline framework/src/main/java/com/github/dbmdz/flusswerk/framework/rabbitmq/Route.java:13]:
```suggestion
  private final List<Topic> topics;
```

File: framework/src/main/java/com/github/dbmdz/flusswerk/framework/rabbitmq/MessageBroker.java [hunk 3]
```diff
@@ -188,7 +193,8 @@ private void provideInputQueues() throws IOException {
   }
 
   private void provideOutputQueues() throws IOException {
-    for (String topic : routingConfig.getOutgoing().values()) {
+    for (String topic :
+        routingConfig.getOutgoing().values().stream().flatMap(List::stream).toList()) {
```
Comment (bitzl) [inline framework/src/main/java/com/github/dbmdz/flusswerk/framework/rabbitmq/MessageBroker.java:197]: Would a helper method like e.g. `Stream<String> RoutingProperties.allOutgoing()` be useful to make this easier to read?

File: integration-tests/src/test/java/com/github/dbmdz/flusswerk/integration/RabbitUtil.java [hunk 4]
```diff
@@ -98,7 +98,8 @@ public void purgeQueues() {
 
   public List<Queue> allQueues() {
     Stream<String> queueNames = routing.getIncoming().stream();
-    queueNames = Stream.concat(queueNames, routing.getOutgoing().values().stream());
+    queueNames =
+        Stream.concat(queueNames, routing.getOutgoing().values().stream().flatMap(List::stream));
```
Comment (bitzl) [inline integration-tests/src/test/java/com/github/dbmdz/flusswerk/integration/RabbitUtil.java:102]: Would a helper method like e.g. `Stream<String> RoutingProperties.allOutgoing()` be useful to make this easier to read?

**Review by bitzl — COMMENTED** [review]
Nice work!

File: framework/src/main/java/com/github/dbmdz/flusswerk/framework/rabbitmq/MessageBroker.java (hunk 3 above)
Comment (schmika) [reply framework/src/main/java/com/github/dbmdz/flusswerk/framework/rabbitmq/MessageBroker.java:197]: That's a very good suggestion, I've added the method. Thank you!

**Review by jbaiter — APPROVED** [review]

**Pull request merged**